PWGJE/FlavourJetTasks/AliAnalysisTaskHFSubstructure.cxx

```diff
@@ -1373,15 +1368,14 @@ Bool_t AliAnalysisTaskHFSubstructure::FillHistograms()
     if (NTruthD==5) fhEvent->Fill(16); 
     if (NTruthD==6) fhEvent->Fill(17);
     }
-    delete Particle_Container; 
```

Comment:
From what I see you take the particle container from the AliAnalysisTaskEmcal via GetParticleContainer. The AliAnalysisTaskEmcal however keeps ownership over the particle containers. This should result in a double delete, and you possibly get into trouble the next time you access the particle container.